README.md

```diff
@@ -1,2 +1,207 @@
 # ea_enterprise
-Collection of containers to run Enroll in development context
+Collection of containers to run the Enroll ecosystem in a development context
+
+## Getting Started
+
+Download and install Docker Desktop.
+
+Clone the ea_enterprise repo:
+```
+git clone https://github.com/ideacrew/ea_enterprise.git
+```
+
+Copy the .env.example to .env and change the values as needed (don't commit .env file)
```

Comment:
This should be updated to `env.dev` instead of `.env`

Reply:
fixed! thanks a lot